src/com/company/reverse/ReverseTest.java

```diff
@@ -0,0 +1,31 @@
+package com.company.reverse;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ReverseTest {
+    Reverse reverse = new Reverse();
```

Comment:
please use access modifiers for class fields

Reply:
Done